examples/advanced/example.js

```diff
@@ -425,7 +424,9 @@ async function initMap(center) {
     destinationLocation = currentDisplayedPlace.geometry.location;
     $("#destination-input").val(currentDisplayedPlace.formatted_address);
 
-    calculateRoute();
+    calculateRoute(() => {
+      $("#routes-container").show(); // Don't show container until route information is retrieved
+    });
```

Comment:
Nit: If this is the only place where we call this, you could just move this container show to the end of the `calculateRoute` function.